src/routes/api/kanban/mockData.ts

```
@@ -0,0 +1,26 @@
+// src/routes/api/kanban/mockData.ts
+export let mockColumns = [
```

ESLint

[error] 'mockColumns' is never reassigned. Use 'const' instead.

(prefer-const)